src/app/api/notification/notification.helpers.ts

```diff
@@ -0,0 +1,60 @@
+import { NotificationTaskActions } from '../core/types/tasks'
+
+/**
+ * Helper function that sets the in-product notification title and body for a given notification trigger
+ * @param actionUser The user's name that triggered this action.
+ */
+export const getInProductNotificationDetails = (actionUser: string, taskName?: string) => {
+  return {
+    [NotificationTaskActions.Assigned]: {
+      title: 'Task was assigned to you',
+      body: `A new task was assigned to you by ${actionUser}. To see details about the task, navigate to the Tasks App below.`,
+    },
+    [NotificationTaskActions.Completed]: {
+      title: 'A client completed a task',
+      body: `A new task was completed by ${actionUser}. You are receiving this notification because you have access to the client.`,
+    },
+    [NotificationTaskActions.Commented]: {
+      title: 'New comment on task',
+      body: `A new comment was left by ${actionUser} on a task where you are set as the assignee. To see details about the task, navigate to the Tasks App below.`,
+    },
+    [NotificationTaskActions.Mentioned]: {
+      title: 'You were mentioned in a task comment',
+      body: `You were mentioned in a comment on task ${taskName} by ${actionUser}. To see details about the task, navigate to the Tasks App below. `,
+    },
+  }
+}
+
+/**
+ * Helper function that sets the notification email details for a given notification trigger
+ * @param actionUser The user's name that triggered this action.
+ * TODO: Right now its the same as in-product details, change this after finalizing email details
+ */
+export const getEmailDetails = (actionUser: string, taskName?: string) => {
```

Comment:
missing jsdoc for param `taskName`

Reply:
Fixed in 3a9dcb86db254fe7b133359ecb68d8b0b1d765be commit